### DISCUSSION TITLE
Can firefox-scripts also be applied to Firefox Android [root]?

### DISCUSSION THREAD

**Discussion by LeeBinder** [xiaoxiaoflood/firefox-scripts#131] [General Support] [answered]
And if so, where would the files need to be copied to? Would the Firefox (or Fennec, Iceraven etc.) apk itself need to be modified (and resigned), too? Or is that not possible in general at this project's current state?

Needless to say, prerequisite is a rooted system.

**Comment by xiaoxiaoflood** [comment, marked as answer]
Hmm, I don't know, probably not. I don't use mobile, I know nothing about Firefox Android. Unless that changes, it won't happen.

**Comment by LeeBinder** [comment]
Gotcha. It's just the way it is. Thanks for replying.
